src/template-analyzer.ts

```diff
@@ -198,19 +198,39 @@ export class TemplateAnalyzer {
    */
   public resolveLocation(
     loc: parse5.Location
-  ): ESTree.SourceLocation | null | undefined {
+  ): ESTree.SourceLocation {
     let offset = 0;
+    let height = 0;
 
     for (const quasi of this._node.quasi.quasis) {
       const placeholder = getExpressionPlaceholder(this._node, quasi);
       offset += quasi.value.raw.length + placeholder.length;
 
+      const i = this._node.quasi.quasis.indexOf(quasi);
+      if (i !== 0) {
+        const expression = this._node.quasi.expressions[i - 1];
+        height += expression.loc.end.line - expression.loc.start.line;
```

Comment:
since `expression.loc` is nullable, we should return `null` i think and reintroduce that in the return type. like a short circuit